app/templates/components/ui-table/cell/cell-name.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if record}}
+  {{record.name}}
```

Comment:
oversight? This does not make sense.

Reply:
Actually I added this template just to fix issue mentioned here - https://github.com/fossasia/open-event-frontend/pull/3198#issuecomment-518772521

But it isn't helping. I used `sesstionType.name` and `track.name` earlier in column in list.js

Reply:
@shreyanshdwivedi Try using extraValuePaths as done in other tables to use other properties of a variable.
https://github.com/fossasia/open-event-frontend/blob/ac210a331d034d9ce1e98380923271656c334f53/app/controllers/admin/messages/list.js#L30

Reply:
No help @kushthedude 
@CosmicCoder96 reverting it to previous state i.e. passing `valuePath     : 'track.name'` and `valuePath     : 'sessionType.name'` in respective columns.